### PR TITLE
Add fasttext 0.1.0 recipe

### PR DESCRIPTION
**Pull request by JungeAlexander**

### CODE REVIEW

File: recipes/fasttext/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  url: https://github.com/facebookresearch/fastText/archive/431c9e2.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - gcc   # [linux]
```
Comment (dougalsutherland) [inline recipes/fasttext/meta.yaml]: You shouldn't need this; the system GCC we use is new enough (according to the docs).

Make sure to include `toolchain` as a build requirement.

File: recipes/fasttext/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - llvm  # [osx]
```
Comment (dougalsutherland) [inline recipes/fasttext/meta.yaml]: This is from `defaults` and probably isn't what you want. Try just using the system compiler on Mac too (with `toolchain` as a build requirement).

File: recipes/fasttext/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  run:
+    - libgcc    # [linux]
```
Comment (dougalsutherland) [inline recipes/fasttext/meta.yaml]: Should be able to drop this, and the whole `run` section.

File: recipes/fasttext/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+test:
+  commands:
+    - fasttext supervised -h
```
Comment (dougalsutherland) [inline recipes/fasttext/meta.yaml]: `-h` annoying returns 1, so you should do something like <code>fasttext supervised -h || [[ $? == 1 ]] &nbsp;# -h returns 1</code>

File: recipes/fasttext/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+about:
+  home: https://fasttext.cc/
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'fastText - Library for efficient text classification and representation learning'
+  dev_url: https://github.com/facebookresearch/fastText
```
Comment (dougalsutherland) [inline recipes/fasttext/meta.yaml]: You could add `doc_url: https://fasttext.cc/docs/en/support.html` too.

File: recipes/fasttext/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,40 @@
+{% set name = "fasttext" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "8c4ba14372d160fe7de902299caac664b7c1af1778601e17ca18ca2081b0e253" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: 431c9e2.tar.gz
```
Comment (dougalsutherland) [inline recipes/fasttext/meta.yaml]: Please change to `{{ name }}-{{ version }}.tar.gz` – this key is confusing, it doesn't refer to the last component of the url but to the local filename that conda will keep in the cache.

File: recipes/fasttext/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  url: https://github.com/facebookresearch/fastText/archive/431c9e2.tar.gz
```
Comment (dougalsutherland) [inline recipes/fasttext/meta.yaml]: `fasttext` did the github release in a very strange way, so this URL is kind of unfortunate. I think for increased maintainability, it'd be better to add `{% set release_name = "431c9e2" %}` under the `set version` line above, and then change this to `https://github.com/facebookresearch/fastText/archive/{{ release_name }}.tar.gz`.

**Comment by JungeAlexander** [comment]
@dougalsutherland great, thank you very much for suggestions! I think I implemented all of them; fingers crossed for the next CI run.

**Comment by JungeAlexander** [comment]
Yes, looks good. Feel free to merge whenever it suits you

**Comment by dougalsutherland** [comment]
Merged. Thanks @JungeAlexander!

**Pull request merged**